README.md

```diff
@@ -1 +1,51 @@
-# group-IV
+# LUNCHER - Bunch your lunch!
+
+If you are hungry, but not sure what to eat, using only one product from you resources you can get few delicious options for your lunch. Easy and fast recipes for every hungry individual.
+
+## Getting Started
+
+This app is designed for every individual who wish to have a good, fast and healthy lunch. With only one ingredient you can get a list of options and choose one that is the best.
+Enjoy.
+
+## Motivation
+
+A Ragnarson Internship 2016 group project: from theory to practice.
+
+## Installation
+
+If you wish to take part in our project feel free to clone it (using terminal):
+
+```
+$ git clone https://github.com/Ragnarson-Internship2016/group-IV
+```
+
+## Development
+
+In terminal, run command
+``` rails server ```
```

Comment:
it should be:

\`\`\`
$ rails server
\`\`\`

or even bin/rails instead of rails as you have binstubs created